### DISCUSSION TITLE
How do I add errors to errors.root.* using the validation library?

### DISCUSSION THREAD

**Discussion by maroKanatani** [react-hook-form/react-hook-form#10019] [Q&A]
I would like to implement form level validation when using a validation library such as yup.
I saw the following discussion and thought it would be a good idea to use `errors.root.*` .
https://github.com/react-hook-form/react-hook-form/discussions/9691

However, I am having trouble with `errors.root.*` not being updated when I create an error with a path of `root.*` during validation.

codesandbox link: https://codesandbox.io/s/async-meadow-8y8zkl?file=/src/App.js

In the above example, in addition to validating each field, I am trying to do a form level error that checks if the values are the same.

By using `createError({ path: "root.formLevelError" })`, I expect the error to go into `errors.root.formLevelError` .
but no error is set.

```js
createError({
     path: "root.formLevelError",
     message: "A and B must be the same number."
})
```

**Comment by Moshyfawn** [comment]
`resolvers` package doesn't support `root` error yet

**Reply by maroKanatani** [reply]
Thank you for your reply.
Will this be supported in the future?
